app_dart/micro_services/results_pipeline/lib/common/response.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+
+/// Class that represents an HTTP response body before it has been serialized.
+@immutable
+abstract class Body {
```

Comment:
Going forward, I'm against reusing `Body`. I'd prefer we use something like `package:http` instead of writing our own classes on top of the `dart:io` calls. This class has slowed my ability to develop features due to it's low level functionality.

Reply:
Very happy to hear this! :) will remove.